.github/workflows/test.yml

```diff
@@ -1,4 +1,4 @@
-name: test
+name: Test
```

Comment:
For consistency, should we also modify the name in `deploy.yml`?

Reply:
Updated 🙏